lib/src/test/kotlin/starknet/utils/DevnetClient.kt

```diff
@@ -405,9 +409,29 @@ class DevnetClient(
         // First two of them come from "scarb build" output, and don't have "command" field
         // Last object is the actual one we want to return
         // Retrieving the last object works in both cases - with one and with few response objects
-        val lines = String(process.inputStream.readAllBytes()).trim().split("\n")
+        val lines = String(process.inputStream.readAllBytes()).trim().split("\n").filter { it != "null" }
         val result = lines.last()
-        return json.decodeFromString(SnCastResponsePolymorphicSerializer, result)
+
+        // Workaround: "command" field is not present in sncast response anymore, hence
+        // we need to put it to JSON synthetically
+        val jsonElement = json.parseToJsonElement(result).jsonObject.toMutableMap()
+        val commandStr = if (command == "account") {
+            command + " " + (
+                args.getOrNull(0)
+                    ?: throw IllegalArgumentException("Missing subcommand for account command")
+                )
+        } else {
+            command
+        }
+        jsonElement["command"] = JsonPrimitive(commandStr)
+
+        // Remove fields with null values
+        jsonElement.entries.removeIf { it.value is JsonNull }
+
+        return json.decodeFromString(
+            SnCastResponsePolymorphicSerializer,
+            json.encodeToString(JsonObject(jsonElement)),
+        )
```

Comment:
note: Technically we should bring it back in `sncast` because it was a breaking change (which we didn't notice 😅 ).